contracts/src/v0.8/upkeeps/EthBalanceMonitor.sol

```diff
@@ -2,8 +2,9 @@
 
 pragma solidity 0.8.6;
 
-import "../ConfirmedOwner.sol";
-import "../interfaces/KeeperCompatibleInterface.sol";
+import "@chainlink/contracts/src/v0.8/KeeperCompatible.sol";
```

Comment:
does this just pull in the latest? if so that could be problematic. Also probably would want to make such a change repo wide for consistency cc @alexroan

Reply:
We should keep these paths relative in the core repo.

This sounds like a DevRel issue that we could resolve in the [external documentation repo](https://github.com/smartcontractkit/documentation/). There are [several](https://docs.chain.link/docs/get-the-latest-price/#solidity) [examples](https://docs.chain.link/docs/get-a-random-number/) of solidity code that we provide for Remix.

Reply:
Ok I think the action items here are we will scrap this PR, and ask devrel to create a new example using the examples Alex provided where we do not specify relative paths (so new example file)